tox.ini

```diff
@@ -27,3 +27,10 @@ commands =
 commands =
     python --version
     nosetests {posargs}
+
+[testenv:flake8]
+description = "Code quality inspection using flake8"
+skip_install = true
+ignore_errors = false
```

Comment:
Isn't that the default?

Reply:
It might be, I tend to be on the verbose side when it comes to configs.